.github/workflows/coverage.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: '14'
+          node-version: 18
```

Comment:
let's file a separate PR for this

Reply:
it's either this or running yarn install with `--ignore-engines`. I prefer the node version update

Reply:
changed to 20

Reply:
we are actually getting test failures here in node 20